service/topology_coordinator.cc

```diff
@@ -2655,27 +2657,25 @@ future<> topology_coordinator::build_coordinator_state(group0_guard guard) {
     std::set<sstring> enabled_features;
 
     // Build per-node state
-    for (const auto& host_id: tmptr->get_normal_token_owners()) {
-        const auto ep = tmptr->get_endpoint_for_host_id_if_known(host_id);
-        if (!ep) {
-            throw std::runtime_error(format("failed to build initial raft topology state from gossip for node with ID {}, as its IP is not known", host_id));
-        }
-        const auto eptr = _gossiper.get_endpoint_state_ptr(*ep);
+    for (const auto* node: tmptr->get_topology().get_nodes()) {
```

Comment:
Shouldn't this filter for `is_member`, to simulate the previous logic?

Reply:
It should, I'll fix it.